content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

```diff
@@ -13,8 +13,8 @@ as a container runtime after v1.20.
 
 **You do not need to panic. It’s not as dramatic as it sounds.**
 
-tl;dr Docker as an underlying runtime is being deprecated in favor of runtimes
```

Comment:
The original is idiomatic, uh, web forum slang.

Reply:
@sftime I'll make it lowercase again.  I'm used to seeing it uppercase in some places.